setup.py

```diff
@@ -1,32 +1,312 @@
+        2/ parse each file in the to_be_parsed_files list:
+            - search each line with one of these patterns:
+                ALIGNAKETC
+                ALIGNAKVAR
+                ALIGNAKBIN
+                ALIGNAKRUN
+                ALIGNAKLOG
+                ALIGNAKLIB
+                ALIGNAKUSER
+                ALIGNAKGROUP
+            - replace found pattern with the value determined for each variable
+        """
+        # Before data files installation ...
+        # ... backup existing files
+        if to_be_installed_files:
+            installation_date = datetime.datetime.now().strftime('%Y-%m-%d_%H-%M-%S')
+
+            print "Checking former existing files..."
+            for dir,file in to_be_installed_files:
+                filename = os.path.join(dir, file)
+                if os.path.isfile(filename):
+                    bkp_file = "%s_%s%s_" % (
+                        os.path.splitext(filename)[0],
+                        installation_date,
+                        os.path.splitext(filename)[1]
+                    )
+                    print " -> backing up file: %s/%s to %s" % (dir, file, bkp_file)
+                    os.rename(filename, bkp_file)
+                    print " -> backed up: %s/%s" % (dir, file)
+
+        # Setuptools install_data ...
+        _install_data.run(self)
+
+        # After data files installation ...
+        # ... parse configuration files to update installation dir
+        if to_be_parsed_files:
+            # Prepare pattern for alignak.cfg
+            to_change = re.compile("|".join(alignak_cfg.keys()))
+
+            print "Parsing files..."
+            for dir,file in to_be_parsed_files:
+                print " -> parsing %s%s" % (dir,file)
+
+                # Parse file
+                for line in fileinput.input(os.path.join(dir, file), inplace=True):
+                    print(to_change.sub(lambda m: alignak_cfg[re.escape(m.group(0))], line))
+
+                # Rename .parse file
+                os.rename(os.path.join(dir, file), os.path.join(dir, file[:-6]))
+                print " -> parsed %s%s" % (dir,file[:-6])
+
+# Get default Alignak paths ...
+alignak_cfg = get_alignak_cfg()
+if not alignak_cfg:
+    sys.exit("Alignak default paths not found!")
+
+print "Alignak config: "
+for path in alignak_cfg:
+    print " %s = %s" % (path, alignak_cfg[path])
+
+# Define installation paths
+# Get Alignak root installation directory
+alignak_etc_path = alignak_cfg['ALIGNAKETC']
+# Get Alignak configuration packs directory
+alignak_modules_path = os.path.join(
+    alignak_cfg['ALIGNAKETC'], 'arbiter', 'modules'
+)
+# Get Alignak libexec directory
+alignak_libexec_path = alignak_cfg['ALIGNAKLIB']
+
+# Build list of all installable package files
+data_files = []
+to_be_parsed_files = []
+to_be_installed_files = []
+for subdir, dirs, files in os.walk(manifest["__pkg_name__"]):
+    for file in files:
+        # Ignore files which name starts with __
```

Comment:
Some __init__.py file because we are in a Python package ... and you have one for the module Python package :)